configs/localization/bmn/README.md

```diff
@@ -70,5 +70,10 @@ You can also test the action detection performance of the model, with [anet_cuhk
 python tools/analysis/report_map.py --proposal path/to/proposal_file
 ```
 
+Notes:
+1. (Optional) You can use the following command to generate a formatted proposal file, which will be fed into the action classifier (e.g: SSN) to get the classification result of proposals.
```

Comment:
What does `formatted proposal file` mean?

Reply:
Any other `action classifier` (e.g. I3D) could be used?

Reply:
Only SSN and PGCN use this `formatted proposal file` as input, and generate the classification result of proposals.

Reply:
Okay, you can put this info into the notes. When one see `action classifier`, he/she may think about TSN etc.

Reply:
OK.